src/entities/Proposal/routes.ts

```diff
@@ -409,11 +410,44 @@ export async function getProposal(req: Request<{ proposal: string }>) {
 }
 
 const updateProposalStatusValidator = schema.compile(updateProposalStatusScheme)
+
+export async function commentProposalUpdateInDiscourse(id: string, caller: string) {
```

Comment:
Some comments about this method:
- this is where the message builder is used, and returns a message depending on the proposal status
- I left some logging so we can test it. For future development I'd like us to consider using a logging library like  https://tslog.js.org/#/ so we can configure the log levels :)
- When an update on discourse fails, for the time being I decided to log it and call it a day. We could implement a retry mechanism

Reply:
- Instead of finding proposals one by one, I'd rather fetch them all at once from the db, with the updated statuses after the job is finished. However, I thought I didn't want to make this PR any longer, so for now it fetches them by one. I hope this is OK.